tests/phase_function_syntax.prm

```diff
@@ -0,0 +1,154 @@
+# This is a copy of latent_heat.prm, meant to test the input
+# syntax of the PhaseFunction object.
```

Comment:
Can you be a bit more specific here and say what exactly is tested? 
Moreover, how is this test conceptually different from latent_heat_viscosity? Wouldn't it make more sense to have a test with more than one compositional field?